### PR TITLE
[metrics] add math equations in ignite.metrics

### LINKED ISSUE #1577: [docs] add equations in `ignite.metrics`
## 🚀 Feature
Add equations in `ignite.metrics` like PyTorch does on loss functions and also we can revisit the equation later if needed :]

### PR DESCRIPTION
**Pull request by ydcjeff**
Fixes #1577 

Description: add math equations in `ignite.metrics`

Check list:
* [ ] New tests are added (if a new feature is added)
* [x] New doc strings: description and/or example code are in RST format
* [x] Documentation is updated (if required)

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -193,7 +193,10 @@
 # -- Options for intersphinx extension ---------------------------------------
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {"https://docs.python.org/3/": None}
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None),
```
Comment (vfdev-5) [inline docs/source/conf.py:197]: I think I've already seen such change in another PR :)

**Review by vfdev-5 — COMMENTED** [review]
Thanks for the PR @ydcjeff !

**Review by vfdev-5 — APPROVED** [review]

File: docs/source/conf.py (hunk 1 above)
Comment (ydcjeff) [reply docs/source/conf.py:197]: Yes, this is needed to render properly. I made the change exactly the same, so there won't be conflicts.
